packages/tools/src/types/SegmentationStateTypes.ts

```diff
@@ -266,9 +270,15 @@ export type SegmentationPublicInput = {
 export type RepresentationPublicInput = {
   segmentationId: string;
   type: Enums.SegmentationRepresentations;
-  options?: {
-    // color lut to use for this representation (optional), it can
-    // be either a colorLUT array or the index of the colorLUT in the state
-    colorLUTOrIndex?: Types.ColorLUT | number;
-  };
+  options?: RepresentationPublicInputOptions;
+};
+
+export type RepresentationPublicInputOptions = {
+  segmentationRepresentationUID?: string;
+  // color lut to use for this representation (optional), it can
+  // be either a colorLUT array or the index of the colorLUT in the state
+  colorLUTOrIndex?: Types.ColorLUT | number;
+  // whether to use polymorphic segmentation utilities to convert
+  // from other representations to this representation
+  polySeg?: boolean;
```

Comment:
hasPolymorphicSeg?  Just from the name, I was thinking it was a polygon segmentation?

Reply:
I'd like to use the naming convention of the 3D Slicer, which is polySEG. You can find it at this link: [PolySeg](https://github.com/PerkLab/PolySeg). Is that alright with you?